language/tools/move-cli/src/sandbox/utils/on_disk_state_view.rs

```diff
@@ -100,7 +100,7 @@ impl OnDiskStateView {
 
     fn get_addr_path(&self, addr: &AccountAddress) -> PathBuf {
         let mut path = self.storage_dir.clone();
-        path.push(format!("0x{}", addr));
+        path.push(format!("{}", addr));
```

Comment:
Why are these two changes to address formatting needed? Is the runtime perhaps doing formatting differently than move expects?